qa/rpc-tests/mempool_nu_activation.py

```diff
@@ -62,7 +62,7 @@ def run_test(self):
         node1_taddr = get_coinbase_address(self.nodes[1])
         node0_zaddr = self.nodes[0].z_getnewaddress('sapling')
         recipients = [{'address': node0_zaddr, 'amount': Decimal('10')}]
-        myopid = self.nodes[1].z_sendmany(node1_taddr, recipients, 1, 0, 'AllowRevealedSenders')
+        myopid = self.nodes[1].z_sendmany(node1_taddr, recipients, 1, 0)
```

Comment:
I can’t quite revert it, because the significant difference there is that the error is now async, which `assert_raises_message` doesn’t support. But I’ll remove the explicit privacy policy.

Reply:
Oh, yeah, just the explicit privacy policy was what I was referring to.